### ISSUE TITLE
Add macOS/Linux binary downloads.

### ISSUE THREAD

**Issue by marcelosousa** [github/semantic#17]
Is there a plan to do a release of binaries? That would be quite helpful for people who just want to try out the tool.

**Comment by patrickt** [comment]
I agree we should do this. Just a matter of figuring out the correct place to put it. Probably a `semantic-0.5.0.0-alpha` release or something.

**Comment by tclem** [comment]
Not quite the same thing, but we are now [publishing docker images](https://github.com/github/semantic/packages/11609).

`docker run docker.pkg.github.com/github/semantic/semantic`
